data/ecosystems/m/multiversx-elrond.toml

```diff
@@ -1164,3 +1164,30 @@ url = "https://github.com/yum0e/my-simple-explanations"
 
 [[repo]]
 url = "https://github.com/zdenal/elrond-amm"
+
+[[repo]]
+url = "https://github.com/multiversx/mx-exchange-docs"
+
+[[repo]]
+url = "https://github.com/multiversx/mx-exchange-service"
+
+[[repo]]
+url = "https://github.com/multiversx/mx-template-nextjs-dapp"
+
+[[repo]]
+url = "https://github.com/wdcurry/mxConsole"
+
+[[repo]]
+url = "https://github.com/PeerMeHQ"
```

Comment:
This warrants it's own sub ecosystem since it is a github org.  Can you spin that one out?

Reply:
Done. Thanks!
:raised_hands: